python-for-beginners/17 - JSON/read_json.py

```diff
@@ -29,8 +31,10 @@
 # According to the documentation for the analyze image function
 # we need to specify the subscription key and the content type
 # in the HTTP header. Content-Type is application/octet-stream when you pass in a image directly
-headers    = {'Content-Type': 'application/octet-stream',
-              'Ocp-Apim-Subscription-Key': subscription_key}
+headers    = {
```

Comment:
`headers = {` 
This might related to indent(maybe some align), fixing or not is on your own.